validator/src/main.rs

```diff
@@ -161,7 +162,10 @@ fn wait_for_restart_window(
         seen_incremential_snapshot |= snapshot_slot_info_has_incremential;
 
         let epoch_info = rpc_client.get_epoch_info_with_commitment(CommitmentConfig::processed())?;
-        let healthy = rpc_client.get_health().ok().is_some();
+        let healthy = match skip_health_check {
+            false => Some(rpc_client.get_health().ok().is_some()),
+            true => None,
+        };
```

Comment:
nit: how about just this?  Then the line on 236 doesn't need to change
```suggestion
        let healthy = skip_health_check || rpc_client.get_health().ok().is_some();
```

Reply:
That was my initial thought, actually ☺️ But I was concerned that I wouldn't pass the review because the variable name/type doesn't fully correspond to its value...